core/lib/zksync_core/src/state_keeper/seal_criteria/criteria/pubdata_bytes.rs

```diff
@@ -79,7 +95,7 @@ mod tests {
         let block_execution_metrics = ExecutionMetrics {
             l2_l1_long_messages: (config.max_pubdata_per_batch as f64
                 * config.close_block_at_eth_params_percentage
-                - 1.0)
```

Comment:
I changed it accordingly to the consts. because it is adding some numbers in criteria itself - which results in the test not passing. I am not fully sure that it is the expected behaviour, but if it is - my fault is that I probably should use const but not hardcode numbers